src/app/api/auth/signout/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextResponse } from 'next/server';
+
+/**
+ * @file /api/auth/signout/route.ts
+ * @description 사용자의 로그아웃 요청을 처리하여 인증 쿠키를 삭제하는 API 라우트 핸들러입니다.
+ * @see /src/shared/hooks/useSignout.ts - 이 API를 호출하는 클라이언트 훅
+ *
+ * @description
+ * 클라이언트의 로그아웃 요청을 받아 서버에 저장된 인증 관련 쿠키를 삭제합니다.
+ *
+ * ### 주요 로직:
+ * 1.  클라이언트로부터 `POST` 요청을 받습니다.
+ * 2.  `NextResponse` 객체를 생성합니다.
+ * 3.  `response.cookies.delete()`를 사용하여 `accessToken`과 `refreshToken` 쿠키를 삭제합니다.
+ * 4.  쿠키가 삭제된 응답을 클라이언트에 반환하여 로그아웃 절차를 완료합니다.
+ *
+ * @returns {Promise<NextResponse>} 쿠키가 삭제된 성공 응답 객체.
+ */
+export async function POST() {
```

Comment:
현재 인중 여부와 상관없이 무조건 로그아웃 응답을 반환하는 구조인데, 의도치 않은 쿠키 삭제 요청이 발생해도 무조건 `ok` 응답을 주는 구조입니다. 선택적으로 아래와 같이 `accessToken`이 있는 요청만 로그아웃이 가능하게 검증 로직을 추가할 수 있을 거 같습니다!

```suggestion
export async function POST(request: NextRequest, ...) {
  const cookies = request.cookies;
  if (!cookies.has('accessToken')) {
    return NextResponse.json({ message: '이미 로그아웃 상태입니다.' }, { status: 400 });
  }
```

Reply:
네 여기는 제안하신 부분 반영해서 수정해볼게요. 이후 브랜치에서 수정할게요~